frontend/src/pages/org/browser-profiles-list.ts

```diff
@@ -76,106 +134,182 @@ export class BrowserProfilesList extends LiteElement {
           )}
         </div>
       </header>
-      <div class="overflow-auto px-2 pb-1">${this.renderTable()}</div>`;
+      <div class="pb-1">${this.renderTable()}</div>`;
   }
 
   private renderTable() {
+    const headerCells = [
+      {
+        sortBy: "name",
+        sortDirection: 1,
+        className: "pl-3",
+        label: msg("Name"),
+      },
+      { sortBy: "url", sortDirection: 1, label: msg("Visited URLs") },
+      { sortBy: "created", sortDirection: -1, label: msg("Created On") },
+      { sortBy: "modified", sortDirection: -1, label: msg("Last Updated") },
+    ];
+    const sortProps: Record<
+      SortValues,
+      { name: string; label: string; className: string }
+    > = {
+      none: {
+        name: "arrow-down-up",
+        label: msg("Sortable"),
+        className: tw`text-xs opacity-0 hover:opacity-100 group-hover:opacity-100`,
+      },
+      ascending: {
+        name: "sort-up-alt",
+        label: msg("Ascending"),
+        className: tw`text-base`,
+      },
+      descending: {
+        name: "sort-down",
+        label: msg("Descending"),
+        className: tw`text-base`,
+      },
+    };
+
+    const getSortIcon = (sortValue: SortValues) => {
+      const { name, label, className } = sortProps[sortValue];
+      return html`
+        <sl-icon
+          class=${clsx(tw`ml-1 text-neutral-900 transition-opacity`, className)}
+          name=${name}
+          label=${label}
+        ></sl-icon>
+      `;
+    };
+
     return html`
-      <btrix-table
-        style="grid-template-columns: [clickable-start] 60ch repeat(2, auto) [clickable-end] min-content; --btrix-cell-padding-left: var(--sl-spacing-x-small); --btrix-cell-padding-right: var(--sl-spacing-x-small);"
-      >
+      <btrix-table class="-mx-3 overflow-x-auto px-3">
         <btrix-table-head class="mb-2">
-          <btrix-table-header-cell class="pl-3">
-            ${msg("Name")}
-          </btrix-table-header-cell>
-          <btrix-table-header-cell>
-            ${msg("Visited URLs")}
-          </btrix-table-header-cell>
-          <btrix-table-header-cell>
-            ${msg("Last Updated")}
-          </btrix-table-header-cell>
+          ${headerCells.map(({ sortBy, sortDirection, label, className }) => {
+            const isSorting = sortBy === this.sort.sortBy;
+            const sortValue =
+              (isSorting && SortDirection.get(this.sort.sortDirection)) ||
+              "none";
+            // TODO implement sort render logic in table-header-cell
+            return html`
+              <btrix-table-header-cell
+                class="${className} group cursor-pointer rounded transition-colors hover:bg-neutral-50"
```

Comment:
Would like to change the hover color to primary, neutral doesn't stand out enough for me and primary is in-line with the hover state for navigation buttons.

```suggestion
                class="${className} group cursor-pointer rounded transition-colors hover:bg-primary-50"
```